backend/RestAPI/Controllers/UsersController.cs

```diff
@@ -109,7 +109,7 @@ public async Task<ActionResult<string>> Authenticate(string email, string passwo
 
             if (token is null)
             {
-                return Unauthorized();
+                return NotFound();
```

Comment:
I change this on purpose. Since on this method we check whether user exists or not, so I think it makes sense that it returns `NotFound`. Also, this way the response won't get picked by interceptor in front-end.